controller/channel-billing.go

```
@@ -165,17 +166,14 @@ func GetResponseBody(method, url string, channel *model.Channel, headers http.He
 	if err != nil {
 		return nil, err
 	}
+	defer res.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `res.Body.Close` is not checked

(errcheck)